### DISCUSSION TITLE
Shapash for CNNs

### DISCUSSION THREAD

**Discussion by Rabia3** [MAIF/shapash#256] [Q&A]
Hi everyone,
As Shapash use LIME and SHAP to explain the AI models, all available examples are using tabular dataset(Limetabularexpaliner). Can we use Shapash to explain CNNs model (using image dataset)?

**Comment by yg79** [comment]
Hi @Rabia3,
Shapash works for tabular use cases. We expect to work on NLP in the near future; we have not yet planned any computer vision features. If you have ideas to share on a feature and want to contribute to Shapash, we can develop it together.
